index.html

```diff
@@ -122,6 +122,541 @@ <h4>LET'S CONNECT</h4>
+                    
+                    <h3 class="articulo-trabajo mobile-only">Multi-Post <br class="mobile-only">Stories</h3>
+
+                    <h3 class="articulo-trabajo desktop-only">Uber Navigation</h3>
+
+                    <ul class="articulo-trabajo empresa flex-row ">
+                        
+                         <li>
+                            
+                            <h4 class="mobile-only">CANOPY</h4>
+
+                         </li>
+
+                         <li>
+                            
+                             <h4 class="desktop-only">Uber</h4>
+
+                         </li>
+
+                         <li>
+                             <img src="media/punto.png">
+                         </li>
+
+                         <li class="desktop-only">
+                            
+                             <p>Lead Developer</p>
+
+                         </li>
+
+                         <li>
+                            
+                            <p class="mobile-only">Back End Dev</p>
+
+                         </li>
+                   
+                         <li>
+
+                            <img src="media/punto.png">
+
+                         </li>
+
+                         <li class="desktop-only">
+                            
+                            <p>2018</p>
+
+                         </li>
+
+                         <li class="mobile-only">
+
+                            <p>2015</p>
+
+                         </li>
+
+                    </ul>
+
+                         <p class="articulo-trabajo trabajo-informacion desktop-only">A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.</p>
+                    
+                         <p class="trabajo-informacion articulo-trabajo mobile-only">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+                     <ul class="articulo-trabajo clase flex-row ">
+                        
+                         <li class="html">
+                            
+                             <p>HTML</p>
+
+                         </li>
+                         
+                         <li>
+                            
+                             <p>CSS</p>
+
+                         </li>
+            
+                         <li>
+                            
+                             <p>JavaScript</p>
+
+                         </li>
+
+                     </ul>
+
+                     <div class="grupo-boton">
+                        
+                         <button class="flex-row articulo-trabajo">See project</button>
+
+                     </div>
+
+                </div>
+
+            </div>
+
+            <div id="imagen"><img src="media/escritoriotrabajo4.png" alt="" class="escritorio-trabajo-foto-izquierda"></div>
+         
+         </div>
+
+
+     </div>
+    
```

Comment:
Kindly implement these changes for the works section of your project :

-  The container of the section should be a section tag to show good HTML semantics😄

Reply:
Done👍